src/app/shell/admin-tools/data/history-log/history-log.component.ts

```diff
@@ -163,4 +174,26 @@ export class HistoryLogComponent implements OnInit, OnDestroy {
     Util.setFromPaginationParam(this.filters, this.currentPage, this.totalAmount);
     this.dispatchProperValue(this.tabIndex, this.filters, searchString);
   }
+
+  private initSubscribeOnEachHistory(): void {
```

Comment:
It looks like we have multiple subscriptions to perform the same action.
Can we combine them using a merge operator or some other option?